packages/ckeditor5-html-support/src/htmlcomment.ts

```diff
@@ -244,9 +237,14 @@ export default class HtmlComment extends Plugin {
  * An interface for the HTML comments data.
  *
  * It consists of the {@link module:engine/model/position~Position `position`} and `content`.
- *
- * @typedef {Object} module:html-support/htmlcomment~HtmlCommentData
- *
- * @property {module:engine/model/position~Position} position
- * @property {String} content
  */
+interface HtmlCommentData {
```

Comment:
```suggestion
export interface HtmlCommentData {
```

It's used by `public` methods.